templates/install_puppet.ps1.erb

```diff
@@ -284,6 +284,20 @@ try {
       Stop-Service $service
     }
   }
+  # Wait for any pxp-agent process still hanging around
+  #
+  # There is a known problem for pxp-agent shutdown: there are cases where after service
+  # shutdown pxp-agent processes are still open. See https://tickets.puppetlabs.com/browse/FM-7628
+  # for more details on the symptoms of this.
+  Write-Log "Waiting for pxp-agent processes to stop"
+  Get-Process -Name "pxp-agent" -ErrorAction SilentlyContinue | ForEach-Object {
+    if ($_) {
+      if (!$_.WaitForExit(120000)){

```

Comment:
What unit is this timeout in? Would be nice to have a comment with human-readable format

Reply:
It's milliseconds: https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.waitforexit?view=netframework-4.7.2#System_Diagnostics_Process_WaitForExit_System_Int32_

I'll add a comment